### DISCUSSION TITLE
How to use MS-DIAL in linux?

### DISCUSSION THREAD

**Discussion by typewritermonkey** [systemsomicslab/MsdialWorkbench#179] [Q&A]
Hi everyone I'm a new Linux user, just wondering how to open MSDIAL in Linux? I already downloaded the Linux version of MSDIAL and extracted the zip file. 

I thought I would just need to double click on the MSDIALconsoleapp but nothing happens when I do that. Is there an equivalent to an .exe file like for the windows version? Or does it not have a GUI and you need to use command line interface?

**Comment by kozo2** [comment]
Hi @typewritermonkey 
We do not provide a GUI version of MS-DIAL for Linux.
For Linux, you can only use MS-DIAL as a command-line application.
Please refer to http://prime.psc.riken.jp/compms/msdial/consoleapp.html for instructions on how to use the command-line application.
